From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 16 Sep 2014 14:19:33 +0100
Subject: nlattr: Avoid ABI change in 3.2.61

Commit bfc5184b69cf ('netlink: rate-limit leftover bytes warning and
print process name') added new #includes, and these changed symbol
hashes on some architectures.  Hide them from genksyms.

--- a/lib/nlattr.c
+++ b/lib/nlattr.c
@@ -12,8 +12,10 @@
 #include <linux/netdevice.h>
 #include <linux/skbuff.h>
 #include <linux/string.h>
+#ifndef __GENKSYMS__
 #include <linux/ratelimit.h>
 #include <linux/sched.h>
+#endif
 #include <linux/types.h>
 #include <net/netlink.h>
 
